lib/cyclopedio/mapping/wikipedia_category_utils.rb

```diff
@@ -0,0 +1,67 @@
+require 'cyclopedio/syntax'
+require 'wiktionary/noun'
+
+module Cyclopedio
+  module Mapping
+    # This class provides methods for processing category names and its parsed trees.
+    class WikipediaCategoryUtils
```

Comment:
Thinking about some better name. Maybe we could implement it as a decorator for Category. Definitely it should not be put back to cyclopedio/wiki.